Sources/Sham/Models/StubRequest.swift

```diff
@@ -93,19 +93,19 @@ public struct StubRequest: Hashable, CustomStringConvertible {
         }
 
         // Include any stubbed response where the scheme matches the incoming URL's scheme or is nil or empty
-        let validScheme = url.scheme == requestURL.scheme || url.scheme.isNilOrEmpty
+        let validScheme = url.scheme.isNilOrEmpty || url.scheme == requestURL.scheme
```

Comment:
What's the reason for changing order of operations?

Reply:
Optimization for the bottom-most query comparison (on an OR operation, it will exit on the first true, meaning it never needs to split the string and sort and all that). With that, consistency across the other operations.